test/integration/targets/win_hyperv_guest/aliases

```diff
@@ -0,0 +1 @@
+shippable/windows/group4
```

Comment:
Unfortunately we won't be able to run these tests on Shippable. Our Windows instances are run on EC2, which doesn't support nested virtualization. Instead, mark these tests as unsupported to prevent them from running in CI:

```suggestion
unsupported
```